pkg/objectupdate/rte/rte_test.go

```diff
@@ -195,3 +206,15 @@ func sliceIndex(sl []string, s string) int {
 	}
 	return -1
 }
+func getSetFromStringList(args []string) map[string]string {
```

Comment:
nit: this is not a set, it's more a map. But not worth resubmitting because of this

Reply:
addressed here: https://github.com/openshift-kni/numaresources-operator/pull/852